### PR TITLE
[CODEOWNERS] Add @tshortli as a code owner for availability analysis …

### PR DESCRIPTION
**Pull request by xedin**
…and textual interfaces

### CODE REVIEW

**Comment by xedin** [comment]
cc @tshortli

**Comment by xedin** [comment]
@swift-ci please smoke test

**Review by tshortli — APPROVED** [review]
Thanks!

**Review by shahmishal — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -76,6 +77,8 @@
 
 # lib
 /lib/AST/                                           @hborla @slavapestov @xedin
+/lib/AST/*Availability*                             @tshortli
```
Comment (AnthonyLatsis) [inline .github/CODEOWNERS:80]: Did you mean to assign Allan to `lib/Sema/*Availability*` as well?

Comment (xedin) [reply .github/CODEOWNERS:80]: Ah yes, thank you!

**Pull request merged**